Repositories/UserRepository.cs

```diff
@@ -82,7 +84,7 @@ public async Task Delete(string id)
         {
             try
             {
-                await GetCollection().UpdateOneAsync<User>(x => x.Id == id.ToString(),
```

Comment:
Just removing redundant ToString() methods... I think we may have switched from Guid to string type and forget to update this?